gpdb/download.go

```diff
@@ -26,6 +28,8 @@ var (
 	rx_gpdb_for_6_n_above = `(greenplum-db-(\d+\.)(\d+\.)(\d+)?(\.\d)-rhel5-x86_64.zip|Greenplum Database ` +
 		`(\d+\.)(\d+\.)(\d+)?(\-beta)?(\.\d)?( (Binary Installer|Installer))?( |  )for ` +
 		`((Red Hat Enterprise|RedHat Enterprise|RedHat Entrerprise) Linux|RHEL).*?(7))`
+	// Open Source release
+	rx_open_source_gpdb = `greenplum-(db|database)-(\d+\.)(\d+\.)(\d+)?(\-beta)?(\.\d)?-rhel7-x86_64.rpm`
```

Comment:
```suggestion
	rx_open_source_gpdb = `greenplum-db-(\d+\.)(\d+\.)(\d+)?(\.\d)?-rhel7-x86_64.rpm`
```

it's true that we have a `greenplum-database-6.0.0-beta.6-rhel6-x86_64.rpm` and `greenplum-db-6.0.0-beta.7-rhel6-x86_64.rpm` but maybe it's better we don't show them to users?

Reply:
Its true for end users, they don't care about a beta release, but support folks usually download the beta version to test different feature of the product before an end users.
Keeping the support team on mind, I believe it's ok to keep the beta flag on the regex.